### ISSUE TITLE
podcast: service.download_info has problem parsing podcast RSS

### ISSUE THREAD

**Issue by yaronyogev** [talkpython/mastering-pycharm-course#23]
I tried to duplicate the code of the podcast example and run it, and got no output.
When I looked into it I saw that ElementTree.fromstring did not work well, and the items that findall created were all empty.

**Comment by mikeckennedy** [comment]
Hi @yaronyogev 

Are you referring to this app here?

https://github.com/talkpython/mastering-pycharm-course/tree/master/demos/projects/podcast

If so, it seems to run for me. Just got it off github and ran it. Got this.

```
Welcome to the talk python info downloader.

Working with total of 199 episodes
94. #94 Guarenteed packages via Conda and Conda-Forge
95. #95 Grumpy: Running Python on Go
96. #96 Exploring Awesome Python
97. #97 Flask, Django style with Flask-Diamond
98. #98 Adding concurrency to Django with Django Channels
99. #99 Morepath: Super Powered Python Web Framework
100. #100 Python past, present, and future with Guido van Rossum
101. #101 Adding a full featured Python environment to  Visual Studio Code
102. #102 Effective Code Reviews
103. #103 Compiling Python through PyLLVM and MongoDB for Data Scientists
104. #104 Game Theory in Python
105. #105 A Pythonic Database Tour
106. #106 Invent your own computer games with Python
107. #107 Python concurrency with Curio
108. #108 MicroPython and Open Source Hardware at Adafruit
109. #109 MongoDB Applied Design Patterns
110. #110 Data Democratization with Redash
111. #111 Pythonic Career Advice and More
112. #112 Geeking out in the golden years
113. #113 Dedicated AI chips and running old Python faster at Intel
114. #114 Empowering developers at the Hidden Genius project
115. #115 Python for Humans projects
116. #116 10 top talks of PyCon 2017 reviewed
117. #117 Functional Python with Coconut
118. #118 Serverless software
119. #119 Python in Engineering
120. #120 Python in Finance
121. #121 Microservices in Python
122. #122 Home Assistant: Pythonic Home Automation
123. #123 Lessons from 100 straight dev job interviews
124. #124 Python for AI research
125. #125 Django REST framework and a new API star is born
126. #126 Kubernetes for Pythonistas 
127. #127 Shipping software to users
128. #128 Pythonic Networks with NAPALM
129. #129 Falcon: The bare-metal Python web framework
130. #130 10 books Python developers should be reading
131. #131 Top 10 machine learning libraries
132. #132 Contributing to open source
133. #133 Productivity for developers
134. #134 Python in Climate Science
135. #135 Capturing human moments with AI and Python
136. #136 Secure code lessons from Have I Been Pwned
137. #137 Design concepts and tips for developers
138. #138 Anvil: All web, all Python
139. #139 Paths into a data science career
140. #140 Level up your Python with #100DaysOfCode challenge
141. #141 Python tricks
142. #142 Automating the web with Selenium and InstaPy
143. #143 Tuning Python Web App Performance
144. #144 Machine Learning at the Large Hadron Collider
145. #145 2017 Python Year in Review
146. #146 Building Alexa Skills with Python and Flask
147. #147 Quart: Flask, but 3x faster
148. #148 Python Book Authors'  Panel Discussion
149. #149 4 Python Web Frameworks, Compared
150. #150 Technical Lessons Learned from Pythonic Refactoring
151. #151 Gradual Typing of Production Applications
152. #152 Understanding and using Python's AST
153. #153 How Python Evolves
154. #154 Python in Biology and Genomics
155. #155 Practical steps for moving to Python 3
156. #156 Python History and Perspectives
157. #157 The Journal of Open Source Software

Process finished with exit code 0
```

Do you maybe have an out of date Python? What do you get for `python3 -V`?

**Comment by yaronyogev** [comment]
Hi Michael,

python3 -V says 3.7.2.

When I run the code under the jumpstart project repository, it does run well.

This code where I'm having the problem was copied bit by bit during the viewing of editor lessons of the course, to simulate the points demonstrated during the lesson. After more research, I found that I miscopied at one point and where it should be
```
item.find('pubDate').text,
```
I typed `pubdate`. Duh.
